backend/src/packages/auth/auth.service.ts

```diff
@@ -16,6 +18,16 @@ class AuthService {
   ): Promise<UserSignUpResponseDto> {
     return this.userService.create(userRequestDto);
   }
+
+  public async verifyLoginCredentials({
+    email, // password, --> should be used for password comparison
```

Comment:
After merging the Sign Up we need to add checking with password

Reply:
Done.